german/the-black-hack.md

```diff
@@ -214,7 +214,7 @@ Wenn Charaktere Waffen verwenden, die nicht bei ihrer Klasse aufgelistet stehen,
 
 ## Zufallsbegegnungen
 
-Ein SL kann alle 15 Minuten in der realen Welt (ihr *haltet* das nach, oder?) einen W4 würfeln. Bei einem Ergebnis von 1-2 taucht innerhalb der nächsten **Minute** (Runde) ein zufällig ermitteltes Wesen oder eine Ablenkung auf.
+Ein SL kann alle 15 Minuten in der realen Welt (ihr *achtet darauf*, oder?) einen W4 würfeln. Bei einem Ergebnis von 1-2 taucht innerhalb der nächsten **Minute** (Runde) ein zufällig ermitteltes Wesen oder eine Ablenkung auf.
```

Comment:
Tja, ist wahrscheinlich so ein Fall, wo eine Wendung, die in meiner Gegend eher geläufig ist, woanders exotisch rüberkommt.

Das mit dem "der SL macht"/"du tust Dinge" in einem Satz finde ich allerdings unschön - das klingt, als würde erst vom SL in der 3. Person geredet und er im selben Satz in der Klammer direkt angesprochen. Da würde ich bei meiner Plurallösung bleiben, mehr in dem Sinne, was in der Gruppe üblich ist.

Vielleicht:

> Der SL sollte alle 15 Minuten, die das Spiel in Echtzeit dauert, einen W4 würfeln (ihr *achtet* doch auf die Zeit, oder?).